### PR TITLE
Break down app manager models Part 1

### PR DESCRIPTION
**Pull request by mkangia**
`app_manager/models.py` is huge. So in attempt to break it down,
I would like to release these changes in smaller/reviewable/quickly-mergable-to-avoid-conflicts PRs.
Here is the first one extracting document schemas related to form action.
The naming i have used is schemas -> document -> form_action
Similarly would use schemas -> indexed -> etc in following PR

### CODE REVIEW

File: corehq/apps/app_manager/schemas/document/form_action.py [hunk 1]
```diff
@@ -0,0 +1,253 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import itertools
+from collections import Counter
+
+from dimagi.ext.couchdbkit import (
```
[the remark this answers is not in the document]
Comment (mkangia) [reply corehq/apps/app_manager/schemas/document/form_action.py]: thanks, oversight.

File: corehq/apps/app_manager/sql_models.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
+from __future__ import absolute_import
```
Comment (orangejenny) [inline corehq/apps/app_manager/sql_models.py]: Non-blocking: I'd have named this something like `release_models` or `limited_release_models`, so it explains what the models are, not their implementation details.

**Review by orangejenny — APPROVED** [review]
hooray

File: corehq/apps/app_manager/sql_models.py (hunk 2 above)
Comment (mkangia) [reply corehq/apps/app_manager/sql_models.py]: good point.

**Comment by mkangia** [comment]
Offlined with @esoergel , need to re-do this differently to maintain git history.
Adding a new PR for similar set of changes

**Pull request closed without merging**